sql/mob_spawn_points.sql

```diff
@@ -50573,8 +50573,8 @@ INSERT INTO `mob_spawn_points` VALUES (17350930,'Furies','Furies',28,-185,-10,48
 INSERT INTO `mob_spawn_points` VALUES (17350931,'Furies','Furies',28,-191,-10,51,127);
 
 -- Petrifying Pair
-INSERT INTO `mob_spawn_points` VALUES (17350933,'Kalamainu','Kalamainu',29,-193,-9,50,95);
-INSERT INTO `mob_spawn_points` VALUES (17350934,'Kilioa','Kilioa',30,-185,-10,42,105);
+INSERT INTO `mob_spawn_points` VALUES (17350933,'Kalamainu','Kalamainu',29,-184,-10,45,95);
```

Comment:
these pos come from retail captures? If not and the captures discord doesn't have any for it, we can get some pretty easily :+1:  if they are from a cap, awesome and ty.

Reply:
they were from retail youtube videos, yes. They were previously set into a tree that would cause them to not agro at all in my findings. Spawn positions are most likely within 1-2 yalms accuracy.

Reply:
video is not what is meant by captures. usually, a video accompanies pictures, but we get precise positional data down to 0.0001 yalms from _packet captures_

The capture discord has guides on this stuff.

Reply:
apologies for not realizing you didn't know the lingo sooner

Reply:
oh no it's worries! I'm trying to learn as fast as I can. sorry if I come off as a little dense lol. just started ~2 weeks ago

Reply:
but no live captures were found in the captures discord.

Reply:
kk, we'll hafta to get some then. positions are among the easiest thing to cap, so it shouldn't take long.

Reply:
kk! Want me to revert to what was set previously? or keep this PR open and wait for them?

Reply:
if it were going to take a long while I'd say change back, but I don't think it'll be long. we can just wait.